src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +16,29 @@
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
```

Comment:
Make sure that the `factory` variable is correctly initialized in the `AbstractDao` class, as it is used here to open a session.